core/test_network/tests/integration/sumeragi_with_mock.rs

```diff
@@ -218,31 +220,12 @@ pub mod utils {
     }
 
     pub mod world {
-        use std::{
-            ops::{Deref, DerefMut},
-            str::FromStr as _,
-        };
+        use std::str::FromStr as _;
 
-        use iroha_core::{prelude::*, tx::Domain, wsv::WorldTrait};
+        use iroha_core::{prelude::*, tx::Domain};
         use iroha_data_model::prelude::*;
         use once_cell::sync::Lazy;
 
-        #[derive(Debug, Clone, Default)]
-        pub struct WithAlice(World);
```

Comment:
1. Bacause `WithAlice` adds aditional steps in `with()` method
2. That's strange. You can try replace `<Network<WithAlice, ...>>` with `<Network<World, ...>>` and see if it changes anything

Anyway there is no `WithAlice` in my changes anymore

Reply:
2. as you can see [here](https://github.com/hyperledger/iroha/pull/2347) all is green because `World` was never actually mocked. It was there just to fulfill some type requirement. So this is why I don't see a reason to mock the `World` at all

Reply:
Hmm, can it be green just because this tests are ignoreed:

```rust
#[ignore = "mock"]
```
?

Although I'm agree. After this changes I can see that we don't need mocking for running integrations tests. Mocking was only done in `sumeragi_with_fault.rs` but was never runned in CI/CD (however I don't know why `cargo test --all-features` skips it).
If we don't need this tests I can remove them and World mocking at all

Reply:
ah, I see now, you're right. These tests are ignored

Reply:
does anyone know why these tests are ignored? if not, we'll need mocking most probably

Reply:
These tests aren't ignored, they are singled-out so we can run them separately from the others. 

With that said, these tests don't actually do anything, so stripping them out could work just as easily.